packages/medusa/src/api/routes/admin/return-reasons/create-reason.ts

```diff
@@ -1,11 +1,17 @@
-import { MedusaError, Validator } from "medusa-core-utils"
-import { defaultRelations, defaultFields } from "./"
+import { IsOptional, IsString } from "class-validator"
+import {
+  defaultAdminReturnReasonsFields,
+  defaultAdminReturnReasonsRelations,
+} from "."
+import { ReturnReasonService } from "../../../../services"
+import { validator } from "../../../../utils/validator"
 
 /**
  * @oas [post] /return-reasons
  * operationId: "PostReturnReasons"
  * summary: "Create a Return Reason"
  * description: "Creates a Return Reason"
+ * x-authenticated: true
  * requestBody:
  *   content:
  *     application/json:
```

Comment:
remember to include the required fields(vaule, label) in the OAS documentation schema as 
```
schema: 
  required: 
    - value
    - label
```